src/modules/kvs/test/kvstxn.c

```diff
@@ -31,6 +31,20 @@ static int test_global = 5;
 /* Use when we do not yet have a root_ref. */
 static const char *ref_dummy = "sha1-508259c0f7fd50e47716b50ad1f0fc6ed46017f9";
 
+static void ktest_finalize (struct cache *cache, kvsroot_mgr_t *krm)
```

Comment:
commit message typo "faiulre" -> "failure" in "kvs: [cleanup] refactor repeated code in tests"

Reply:
Thanks!  I was hoping you'd have a look at those changes.  I'll fix the typo and force push in a few minutes.